frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -65,8 +91,8 @@ public boolean unregisterModel(String modelName) throws WorkerInitializationExce
             return false;
         }
 
-        model.setMinWorker(0);
-        model.setMaxWorker(0);
+        model.setMinWorkers(0);
+        model.setMaxWorkers(0);
```

Comment:
We should remove these two parameters and change it to "numWorkers". I don't see a point in having two values when we support only scaling to a particular level.